diffkemp/simpll/DebugInfo.h

```diff
@@ -52,10 +52,6 @@ class DebugInfo {
         calculateMacroAlignments();
         collectLocalVariables(CalledFirst, LocalVariableMapL);
         collectLocalVariables(CalledSecond, LocalVariableMapR);
-        // Remove calls to debug info intrinsics from the functions - it may
-        // cause some non-equalities in FunctionComparator.
```

Comment:
Make sure that the errors mentioned in this comment do not occur - add skipping of comparison of `llvm.dbg.*` instructions into `DifferentialFunctionComparator::cmpBasicBlocks`.

Reply:
Sorry, I forgot to add it. This caused the failure of regression tests. Already fixed.